src/webhooks/pubsub/slackScores.ts

```diff
@@ -26,7 +26,10 @@ export const triggerSlackScores = async (
       const triagedEvents = issueTriageEvents.filter(
         (issue) => issue.is_triaged
       );
-      const score = triagedEvents.length / issueTriageEvents.length;
+      const score =
+        issueTriageEvents.length === 0
+          ? -1
```

Comment:
nit: Still think this is much clearer as an Optional type, rather than a special number you have to check against (ex: not obvious from the type that -1 is the only special number). But not going to block the PR on it, either way works.

Reply:
The reason I didn't change it to be an Optional type is because it introduces some more complex logic to the sorting function that is used later. Otherwise, there needs to be some sort of conditional in the sorting logic specifically for when the variable is undefined. While, if we have a special number 0 for the 0/0 case, the team will be placed at the bottom of the scoreboard automatically. I will write a comment here so that it's more clear.